### PR TITLE
Handle exceptions when `cpu_total` is zero

### PR DESCRIPTION
**Pull request by ivasic**
Ref #610 

I wasn't able to run tests from `cpu_linux_test.py` for some reason, if anyone is willing to help with that I can include a simple test as well.

### CODE REVIEW

**Comment by basepi** [comment]
Hmm, as far as I can tell there are no non-standard requirements to run those tests. Did you install [the base test requirements](https://github.com/elastic/apm-agent-python/blob/d0f2a392fb52d0e09d92aa7036a0353e819ba2f1/tests/requirements/requirements-base.txt)? If so, what error are you seeing when you try to run those tests?

**Comment by ivasic** [comment]
@basepi no errors were thrown, the tests from `cpu_linux_tests.py` were just skipped but I did some digging and figured out that they only run on Linux (I was running them on macOS). Duh... :)  

Just added a simple test that exposes `ZeroDivisionError` without the previous fix

File: tests/metrics/cpu_linux_tests.py [hunk 1]
```diff
@@ -155,3 +155,27 @@ def test_mem_free_from_memfree_when_memavailable_not_mentioned(tmpdir):
...
+    metricset = CPUMetricSet(
+        MetricsRegistry(0, lambda x: None),
+        sys_stats_file=proc_stat,
+        process_stats_file=proc_stat_self,
+        memory_stats_file=proc_meminfo,
+    )
+    data = next(metricset.collect())
+
+    cpu_total_expected = 0
+    assert data["samples"]["system.cpu.total.norm.pct"]["value"] == cpu_total_expected
```
Comment (beniwohli) [inline tests/metrics/cpu_linux_tests.py]: How about also asserting on `system.process.cpu.total.norm.pct` being `0` here?

Comment (ivasic) [reply tests/metrics/cpu_linux_tests.py]: Makes sense! Thanks for the tip

File: tests/metrics/cpu_linux_tests.py [hunk 2]
```diff
@@ -155,3 +155,28 @@ def test_mem_free_from_memfree_when_memavailable_not_mentioned(tmpdir):
...
+    assert data["samples"]["system.process.cpu.total.norm.pct"]["value"] == cpu_total_expected
```
[the remark this answers is not in the document]
Comment (ivasic) [reply tests/metrics/cpu_linux_tests.py]: Right, let's see if the update helps

**Comment by basepi** [comment]
@ivasic I released 5.2.2 with this fix: https://github.com/elastic/apm-agent-python/releases/tag/v5.2.2

Thank you again for the contribution!

**Pull request merged**